packages/web-components/src/components/va-segmented-progress-bar/test/va-segmented-progress-bar.e2e.ts

```diff
@@ -83,3 +85,139 @@ describe('va-segmented-progress-bar', () => {
     expect(analyticsSpy).not.toHaveReceivedEvent();
   });
 });
+
+
+
+describe('v3 va-segmented-progress-bar', () => {
+  it('renders', async () => {
+    const page = await newE2EPage({
+      html: '<va-segmented-progress-bar current="3" total="6" uswds></va-segmented-progress-bar>',
+    });
+    const element = await page.find('va-segmented-progress-bar');
+    expect(element).toEqualHtml(`
+      <va-segmented-progress-bar class=\"hydrated\" current=\"3\" total=\"6\" uswds=\"\">
+        <mock:shadow-root>
+          <div aria-label=\"progress\" class=\"usa-step-indicator\">
+            <ol class=\"usa-step-indicator__segments\">
+              <li class=\"usa-step-indicator__segment usa-step-indicator__segment--complete\"></li>
+              <li class=\"usa-step-indicator__segment usa-step-indicator__segment--complete\"></li>
+              <li class=\"usa-step-indicator__segment usa-step-indicator__segment--current\"></li>
+              <li class=\"usa-step-indicator__segment\"></li>
+              <li class=\"usa-step-indicator__segment\"></li>
+              <li class=\"usa-step-indicator__segment\"></li>
+            </ol>
+            <div class=\"usa-step-indicator__header\">
+              <h4 class=\"usa-step-indicator__heading\">
+                <span class=\"usa-step-indicator__heading-counter\">
+                  <span class=\"usa-sr-only\">
+                    Step
+                  </span>
+                  <span class=\"usa-step-indicator__current-step\">
+                    3
+                  </span>
+                  <span class=\"usa-step-indicator__total-steps\">
+                    of 6
+                  </span>
+                </span>
+                <span class=\"usa-step-indicator__heading-text\"></span>
+              </h4>
+            </div>
+          </div>
+        </mock:shadow-root>
+      </va-segmented-progress-bar>
+    `);
+  });
+
+  it('passes an axe check', async () => {
```

Comment:
For the tests that are specific to `uswds`, can you update the name of the test so that it mentions that? ie. "uswds passes an axe check"

My thinking here is because I'm expecting a lot of "duplicate" tests except one is for v1 and one being for v3.

Reply:
Updated the tests to specify uswds